paparazzi/src/main/java/app/cash/paparazzi/PaparazziSdk.kt

```diff
@@ -303,7 +303,8 @@ public class PaparazziSdk @JvmOverloads constructor(
         }
       }
     } finally {
-      viewGroup.removeView(modifiedView)
+      // Remove original view from parent, even if there aren't render extensions applied
+      (view.parent as ViewGroup).removeView(view)
```

Comment:
nit: in this case any views created by a render extension will not be removed. Is this something we should case about?

Reply:
Yeah I think you need to do both still. Remove the outermost view from the view group, and remove the user view from it's parent. These might be the same so you could make it conditional on `view !== modifierView`.